cpp.mk

```diff
@@ -110,6 +110,7 @@ debug_$1 : $${BUILD_DIR}/bin/$1
 # Valgrind arguments for $1
 $1_VALGRIND_ARGS += --gen-suppressions=all --leak-check=full --track-origins=yes --malloc-fill=cc --free-fill=df
 
+.PHONY: $${BUILD_DIR}/$1/valgrind_output.xml
```

Comment:
Good point - I've changed this (and made the dependency of valgrind_check $${BUILD_DIR}/bin/$1